assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -193,6 +194,49 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
     },
     [wpApi, dataAdapter]
   );
+  const createStoryFromTemplate = useCallback(
+    async (template) => {
+      try {
+        const { createdBy, pages, title, version } = template;
+        const copyTemplateTitle = sprintf(
+          /* translators: %s: template title */
+          __('%s (Copy)', 'web-stories'),
+          title
+        );
+
+        const storyPropsToSave = await getStoryPropsToSave({
+          story: {
+            title: copyTemplateTitle,
```

Comment:
According to Slack, the title should be left empty.

Reply:
Thanks for the eyes on this! If I don't include a title I get a console warning: 
<img width="854" alt="Screen Shot 2020-06-04 at 8 56 17 AM" src="https://user-images.githubusercontent.com/10720454/83779984-725cf080-a641-11ea-8448-6009bb7ca56f.png">

That said, it's also giving me warnings about including an author and metadata.fallbackPoster. 

Should I not worry about these warnings? Is there something we could adjust for on what propTypes are required in `getStoryPropsToSave` or potential fallbacks?

Reply:
Apologies for seeing this so late.

You should pass an empty string then, or make the prop types less strict.